CHANGELOG.md

```diff
@@ -1,8 +1,40 @@
 ## Change Log
 
-### v3.1.0-preview10 - 2020-06-09
+### v3.1.0-preview10 - 2020-08-12
```

Comment:
I believe that we've already released v3.1.0-preview10, so shouldn't the new stuff be under v3.1.0-preview11?